crates/sncast/src/response/structs.rs

```diff
@@ -151,3 +151,35 @@ pub struct VerifyResponse {
 }
 
 impl CommandResponse for VerifyResponse {}
+
+pub trait OutputLink {
```

Comment:
This looks like a better use case for a plain function

Reply:
I have simplified this trait and moved its definition to the module